packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -1,243 +1,174 @@
 @import "../../helpers";
-.sf-header-navigation-item {
-  position: var(--header-navigation-item-position, relative);
+
+.sf-header {  
+  position: fixed;  
+  width: 100%;
+  margin: 0 auto;
   display: flex;
-  flex: var(--header-navigation-item-flex, 0);
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  padding: var(--spacer-2xs);
```

Comment:
question: make sure it doesnt cause the page to scroll horizontally (padding on width:100% can cause such issues)

Reply:
It looks okay, I can see no scrolling.